charts/scalardb-mcp-server/templates/scalardb-mcp-server/deployment.yaml

```
@@ -0,0 +1,98 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "scalardb-mcp-server.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "scalardb-mcp-server.labels" . | nindent 4 }}
+spec:
+  {{- if gt (int .Values.scalardbMcpServer.replicaCount) 1 }}
+  {{- fail "Multiple replicas is currently not supported. Please set replicaCount to 1." }}
+  {{- end }}
+  replicas: {{ .Values.scalardbMcpServer.replicaCount }}
```

```suggestion
  replicas: 1
```
If users cannot choose other values than `1` from the specification or limitation perspective, I think we can simply set `1` as a hard-coded value.

Do you have any special reason why users can set other values in `values.yaml`, and this chart validates it on the template side?

If there is no special reason, I think it would be better to keep the logic simple first. (If we need, we can add such options and validations later.)